src/main/java/com/ericsson/ei/erqueryservice/ERQueryService.java

```diff
@@ -100,14 +104,22 @@ private ResponseEntity sendRequestToER(String eventId, SearchOption searchOption
 
     private void prepareRequest(String eventId, SearchOption searchOption, int limit,
             int levels, boolean tree) throws IOException, URISyntaxException {
+        String shallow="";
         final SearchParameters searchParameters = getSearchParameters(searchOption);
+        if (shallowParameter == null || shallowParameter.isEmpty()) {
+            shallow ="true";
```

Comment:
Has it been discussed what default value to use for shallow flag.
I guess this need to be documented in Markdown documentation.